gdnative-core/src/core_types/geom/plane.rs

```diff
@@ -20,4 +21,262 @@ impl Plane {
     pub fn from_sys(c: sys::godot_plane) -> Self {
         unsafe { std::mem::transmute::<sys::godot_plane, Self>(c) }
     }
+
+    /// Creates a new `Plane` from the ['Vector3'](./type.Vector3.html) normal and the distance from the origin.
+    #[inline]
+    pub fn new_with_normal(normal: Vector3, d: f32) -> Plane {
+        Plane {
+            normal: normal,
+            d: d,
+        }
+    }
+
+    /// Creates a new `Plane` from four floats.
+    /// a, b, c are used for the normal ['Vector3'](./type.Vector3.html).
+    /// d is the distance from the origin.
+    #[inline]
+    pub fn new_with_reals(a: f32, b: f32, c: f32, d: f32) -> Plane {
+        Plane {
+            normal: Vector3::new(a, b, c),
+            d: d,
+        }
+    }
+
+    /// Creates a new `Plane` from three [`Vector3`](./type.Vector3.html), given in clockwise order.
+    #[inline]
+    pub fn new_with_vectors(a: Vector3, b: Vector3, c: Vector3) -> Plane {
```

Comment:
`from_points` would be a name that fits more into Rust conventions.

Also, how does this handle the case where all points are collinear, assuming that this is a constructor that creates a `Plane` that contains all three points? Maybe it could be documented?